### DISCUSSION TITLE
Possible to use WMS source with elevation layer?

### DISCUSSION THREAD

**Discussion by alienatorZ** [iTowns/itowns#1940] [Q&A] [answered]
My geoserver will not serve out application/bil format from gwc and WMTS. The only way I see to get it is with WMS. Is it possible to use a WMSSource with elevation layer?I am trying it and it is not working. 

Thanks!

**Comment by jailln** [comment, marked as answer]
Hi,
Yes it is possible, see https://github.com/iTowns/itowns/blob/master/examples/3dtiles_25d.html#L59-L77 for instance.

**Reply by alienatorZ** [reply]
Thank you for that example. Does iTowns assume 32bit BigEndian format?

**Reply by jailln** [reply]
Did it work?

I would need to give it a better look but it seems that the endianenss used is the one of the system (so most likely littleEndian) since we create a `Float32Array` of the parsed buffer and then create a [DataTexture](https://threejs.org/docs/#api/en/textures/DataTexture) if your data is in the `image/x-bil;bits=32` format (see [here](https://github.com/iTowns/itowns/blob/master/src/Provider/Fetcher.js#L138)). Why?

**Reply by alienatorZ** [reply]
I was trying to configure geoserver DDS plugin which allows for choosing bits and endianness.

Thanks!
